ops/staging/main.tf

```diff
@@ -0,0 +1,104 @@
+locals {
+  project = "prime"
+  name    = "simple-report"
+  env     = "stg"
+  management_tags = {
+    prime-app      = "simple-report"
+    environment    = local.env
+    resource_group = "${local.project}-${local.name}-${local.env}"
+  }
+}
+
+module "simple_report_api" {
+  source = "../services/app_service"
+  name   = "${local.name}-api"
+  env    = local.env
+
+  instance_tier = "Standard"
+  instance_size = "S1"
+
+  resource_group_location = data.azurerm_resource_group.rg.location
+  resource_group_name     = data.azurerm_resource_group.rg.name
+
+  docker_image_uri = "DOCKER|simplereportacr.azurecr.io/api/simple-report-api-build:${var.acr_image_tag}"
+  key_vault_id     = data.azurerm_key_vault.global.id
+  tenant_id        = data.azurerm_client_config.current.tenant_id
+
+  app_settings = {
```

Comment:
Can these settings be moved into the app_service module? Seems like it might be a bit cleaner.

Reply:
we could move the ones that are the same across environments sure